src/Presentation/Nop.Web/Areas/Admin/Factories/ShoppingCartModelFactory.cs

```diff
@@ -170,6 +170,9 @@ public virtual ShoppingCartItemListModel PrepareShoppingCartItemListModel(Shoppi
                     //fill in model values from the entity
                     var itemModel = item.ToModel<ShoppingCartItemModel>();
 
+                    //set product name since it does not survive mapping
+                    itemModel.ProductName = item?.Product?.Name;
+
```

Comment:
This mapping is better to add a couple of lines below, where other fields are added.

Reply:
Wouldn't it be easier to change the mapping ?

```
CreateMap<ShoppingCartItem, ShoppingCartItemModel>()
	.ForMember(model => model.Store, options => options.Ignore())
	.ForMember(model => model.AttributeInfo, options => options.Ignore())
	.ForMember(model => model.UnitPrice, options => options.Ignore())
	.ForMember(model => model.UpdatedOn, options => options.Ignore())
	.ForMember(model => model.ProductName, options => options.MapFrom(x => x.Product != null ? x.Product.Name : string.Empty))
	.ForMember(model => model.Total, options => options.Ignore());
```

That's what Automapper is meant to do.